### PR TITLE
bazarr: 1.0.1

### PR DESCRIPTION
**Pull request by smaarn**
_Motivation:_  Upgrade bazarr to version 1.0.1

More details for the changelog can be found here: https://github.com/morpheus65535/bazarr/releases/tag/v1.0.1

Corresponding github actions binaries expanded and published here: https://github.com/smaarn/spksrc/releases/tag/bazarr-1.0.1-rc1

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Review by publicarray — APPROVED** [review]
LGTM

**Review by th0ma7 — APPROVED** [review]
LGTM as well.

File: spk/bazarr/Makefile [hunk 1]
```diff
@@ -11,10 +11,7 @@ DEPENDS = cross/$(SPK_NAME)
 DEPENDS += cross/unrar
 DEPENDS += cross/numpy
 DEPENDS += cross/webrtcvad
-DEPENDS += cross/gevent
-DEPENDS += cross/gevent-websocket
```
Comment (th0ma7) [inline spk/bazarr/Makefile:15]: Is `gevent` no longer needed?

Comment (publicarray) [reply spk/bazarr/Makefile:15]: Apparently so https://github.com/morpheus65535/bazarr/releases

Comment (th0ma7) [reply spk/bazarr/Makefile:15]: thnx

**Comment by publicarray** [comment]
@smaarn  Thank you for your contribution!

**Pull request merged**